### DISCUSSION TITLE
可否增加自定义刮削源

### DISCUSSION THREAD

**Discussion by lostiv** [metatube-community/jellyfin-plugin-metatube#27] [General]
老一些的片子刮削不到了

**Comment by xjasonlyu** [comment]
举个番号例子？

**Reply by lostiv** [reply]
我的问题，封面图片刮削和重新识别不一样。重新识别更准确，如snis673

**Reply by xjasonlyu** [reply]
重新刮削前后是怎么样的，能截图吗？

**Reply by fjiang423** [reply]
+1, 还有很多问题通过自定义刮削源，或者定义刮削源优先顺序可以解决， 比如：

* 默认源FANZA 识别的时候经常空缺演员，如老片子XV-827 ，新一点的如WANZ-593。标题，介绍，截图都能识别但是演员总是空缺，需要手动识别，选择如AVWIKI的源来补完。
* 有一些老片子像DVAJ-184，截图经常空缺不知道是不是FANZA的原因。希望插件或者后端可以在识别出片源但是截图，封面，演员空缺的时候可以选择备用源来补充

**Reply by xjasonlyu** [reply]
1. 可以通过插件设置中的开启真实演员姓名解决。
2. 这应该不是FANZA的问题。
